conftest.py

```diff
@@ -205,6 +206,16 @@ def pytest_collection_modifyitems(config, items):
             item.add_marker(skip_unit)
 
 
+def pytest_itemcollected(item):
+    # Customize pytest-mpl decorator to add sensible defaults
+
+    mpl_marker = item.get_closest_marker("mpl_image_compare")
```

Comment:
Can you explain this?  Why is tolerance 0.5?  What does this do?

Reply:
Here I'm providing arguments to be passed by default to `@pytest.marker.mpl_image_compare` so they don't have to be repeated.
I went with 0.5, which is [stricter than the default](https://github.com/matplotlib/pytest-mpl#tolerance), because I figured we might want to be extra careful at first since this is a big leap of a transition.